### ISSUE TITLE
"Click" sound between segments

### ISSUE THREAD

**Issue by fraxool** [rFlex/SCRecorder#257]
This is basically a copy of issue #185. I still can't find a solution for this one.

I am trying to add a "fade" effect on audio of each segments of my video but for me, it doesn't work. Can anyone help me or share his fix for this issue ?

Thanks :+1:

**Comment by bishtpradeep** [comment]
facing the same issue. Click sound between multiple segments.

**Comment by mitchellporter** [comment]
I haven't noticed this issue but then again my ears are bad :P 

Is it really that noticeable? What devices and iOS versions are you guys running?

**Comment by bishtpradeep** [comment]
9.2, 6s+,5s. It is very noticeable.

**Comment by mitchellporter** [comment]
@bishtpradeep Does this happen with both the front and back cameras? I've been playing with this library for several weeks now and have never experiences the "click" or "plop" sound, but it still worries me.

**Comment by bishtpradeep** [comment]
Yes. it happens with both cameras on 6s+ [don't have 5s right now to test].
You can reproduce it quickly by recording in short duration segments
around  1 second each.

**Comment by fraxool** [comment]
For me, even if the segment is longer than one second, it happens. I'm using an iPhone 6 with iOS 9.2.

**Comment by mitchellporter** [comment]
Ok, after paying closer attention I know hear the click / plop sound whenever a new segment begins to a play. I'm using an iPhone 6 on iOS 9.2
